### PR TITLE
updates

### PR DESCRIPTION
**Pull request by sgesin**
[@W-7380799](https://gus.lightning.force.com/a07B0000008BArAIAW)

### CODE REVIEW

**Review by smorgan11 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -175,7 +176,20 @@ You can create up to 5,000 custom labels for your organization, and they can be
...
-To update the Custom Labels metadata, go to the `force-app/main/default/labels directory`. The labels are available in the `CustomLabels.labels-meta.xml` file. For an example of a CustomLabels definition, see the [Metadata API Dev Guide](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customlabels.htm).
+To update the Custom Labels metadata, go to the `force-app/main/default/labels` directory. The labels are available in the `lightning.labels-meta.xml` file. For an example of a CustomLabels definition, see the [Metadata API Dev Guide](https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_customlabels.htm).
+
+Add labels for your custom components in the `lightning.labels-meta.xml` file.
+
+```xml
+<labels>
+    <categories>lightning</categories>
+    <fullName>MyCard_cardTitle</fullName>
+    <language>en_US</language>
```
Comment (dianawidjaja) [inline README.md:187]: Do we have guidelines for language values? Maybe something like "Use an ISO language code to describe your label's language. For more information, see https://www.iso.org/iso-639-language-codes.html"?
@sgesin , not sure what we should point people to.

File: README.md [hunk 2]
```diff
@@ -245,12 +259,21 @@ sfdx force:source:push
 
 ### Testing
 
-Tests should be added to the `__tests__` subfolder of the component and typically have a file name that ends with `.spec.js`. The [Jest JavaScript Testing Framework](https://jestjs.io/docs/en/getting-started) is used to author and run component tests.
+If you modify any base component recipes or add your own components, you'll want to run tests. We use the [Jest JavaScript Testing Framework](https://jestjs.io/docs/en/getting-started) to author and run component tests.
```
Comment (dianawidjaja) [inline README.md]: you'll want to run tests -> run tests on your changes.

**Review by dianawidjaja — APPROVED** [review]
Looks great! I added a few comments to clarify the `language` tag values and a minor phrasing suggestion.

File: README.md (hunk 1 above)
Comment (smorgan11) [reply README.md:187]: The Metadata API Dev Guide that we link to just above this mentions this, and links to https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_translations.htm#translation_languages

**Pull request merged**